src/pipecat/services/whisper.py

```diff
@@ -26,18 +27,219 @@
 
 
 class Model(Enum):
-    """Class of basic Whisper model selection options"""
+    """Class of basic Whisper model selection options.
 
+    Available models:
+        Multilingual models:
+            TINY: Smallest multilingual model
+            BASE: Basic multilingual model
+            MEDIUM: Good balance for multilingual
+            LARGE: Best quality multilingual
+            DISTIL_LARGE_V2: Fast multilingual
+
+        English-only models:
+            DISTIL_MEDIUM_EN: Fast English-only
+    """
+
+    # Multilingual models
     TINY = "tiny"
     BASE = "base"
     MEDIUM = "medium"
     LARGE = "large-v3"
     DISTIL_LARGE_V2 = "Systran/faster-distil-whisper-large-v2"
+
+    # English-only models
     DISTIL_MEDIUM_EN = "Systran/faster-distil-whisper-medium.en"
 
 
+def language_to_whisper_language(language: Language) -> Optional[str]:
+    """Maps pipecat Language enum to Whisper language codes.
+
+    Args:
+        language: A Language enum value representing the input language.
+
+    Returns:
+        str or None: The corresponding Whisper language code, or None if not supported.
+
+    Note:
+        Only includes languages officially supported by Whisper.
+    """
+    language_map = {
```

Comment:
It seems that all the languages we need to pass to Whisper only support the first two-letter abbreviation and do not support the country.
Couldn't we just create a generic function to extract the first two characters instead of using this map ?

Reply:
Makes sense. I like the idea of being explicitly about which languages  Whisper actually supports. 👍

Reply:
Makes sense. I like the idea of being explicitly about which languages  Whisper actually supports. 👍